examples/Paris_1872_Alexis/Paris_1872_Alexis.xml

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="../../TEI_documentation/openStemmata.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="../../TEI_documentation/openStemmata.rng"" type="application/xml"
+	schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+  <teiHeader>
+      <fileDesc>
+         <titleStmt>
+            <title type="standard">Stemma of</title>
+            <title type="variable">Chanson de saint Alexis</title>
+            <author>Paris, Gaston and Pannier, Léopold</author>
+            <respStmt>
+               <resp>contributed to OpenStemmata by</resp>
+               <persName>Jean-Baptiste Camps</persName>
+               <idno type="ORCID" n="0000-0003-0385-7037"/>
+            </respStmt>
+         </titleStmt>
+         <publicationStmt>
+            <authority>
+               <name>Openstemmata</name>
+               <ptr target="https://openstemmata.github.io/"/>
+            </authority>
+            <availability>
+               <licence n="cc.by" target="https://creativecommons.org/licenses/by/4.0"/>
+            </availability>
+         </publicationStmt>
+         <notesStmt>
+            <note>Deprecated stemma (see Zufferey’s for a more recent one)</note>
+         </notesStmt>
+         <sourceDesc>
+            <bibl type="book">
```

Comment:
Wondering if we could get away with `biblStruct` or if we are missing information ?

Something like this (Guidelines) example

```xml
<biblStruct>
 <monogr>
  <author>Blain, Virginia</author>
  <author>Clements, Patricia</author>
  <author>Grundy, Isobel</author>
  <title>The Feminist Companion to Literature in English: women writers from the middle ages
     to the present</title>
  <edition>first edition</edition>
  <imprint>
   <publisher>Yale University Press</publisher>
   <pubPlace>New Haven and London</pubPlace>
   <date>1990</date>
  </imprint>
 </monogr>
</biblStruct>
```